src/AudioUnit/AudioUnit.cs

```diff
@@ -352,6 +356,50 @@ public enum AudioUnitClumpID // UInt32 in AudioUnitParameterInfo
 		System = 0
 	}
 
+	public enum AUParameterEventType : uint
+	{
+		Immediate = 1,
+		Ramped = 2,
+	}
+
+	[StructLayout (LayoutKind.Sequential)]
+	public struct AudioUnitParameterEvent
+	{
+		public uint scope;
```

Comment:
Public members should be properly cased, not lower-cased (this goes for all the public fields in this structure and its inner structures).